gizmo.py

```diff
@@ -0,0 +1,55 @@
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+
+    def speak(self):
+        print (self.name)
+
+def hello(name, country = "finland"):
+    print("Hello ",name,", how are things in ",country,"?")
+
+def spell(woord):
+    teller = 1
+    for i in woord:
+        print(i, end='')
+        if teller < len(woord):
+            print(".", end='')
+        teller += 1
+    print()
+
+def relative_path(identifiers):
+    i = 0
+    for identifier in identifiers:
+        identifiers[i] = "./subjects/mock_recording_" + identifier + ".rec"
```

Comment:
This works. But I advise you to check out Python's f-string functionality. This is a very useful feature of the Python language that is worth your time checking out:  https://realpython.com/python-f-strings